autoendpoint/src/routes/registration.rs

```diff
@@ -192,6 +192,12 @@ pub async fn get_channels_route(
     }
     let channel_ids = db.get_channels(&uaid).await?;
 
+    app_state
+        .metrics
+        .count_with_tags("busi.channel_count", channel_ids.len() as u64)
```

Comment:
I don't see how count gets us per uaid stats?

Reply:
We are doing a count for each UAID (since it's done as part of the daily check-in). I expect that the count will be somewhat "noisy" in the way that a number of our metrics are, however the approximate mean should be fairly easy to discern when displayed on a graph. Likewise, significant outliers should also be noticeable, although, I presume specifying the UAID would add too much cardinality. We may be able to later refine that by logging UAIDs that fall outside of the mean with an appropriate buffer, but for now, we don't know what that might be.